### ISSUE TITLE
Apply accessibility guidelines to splink documentation

### ISSUE THREAD

**Issue by mamonu** [moj-analytical-services/splink#642]
- [ ] Change the current default font to an accessible font that follows [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/#distinguishable).There is [functionality](https://squidfunk.github.io/mkdocs-material/setup/changing-the-fonts/) to easily do this with mkdocs

- [ ] The visual presentation of [text](https://www.w3.org/TR/WCAG21/#dfn-text)  needs to have a [contrast ratio](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) of at least 4.5:1 on all colour palettes (light/dark)

**Comment by mamonu** [comment]
additionally there is a way to automatically test for accessibility on websites via the node.js package `Pa11y`.
Mostly everything passes.
Running it on one of the notebook tutorials however gives

```

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > div:nth-child(8) > main > div > div:nth-child(3) > article > div:nth-child(15) > div:nth-child(2) > div > div > div > img
   └── <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA6QAAAh6CAYAAACbjNB4AAAAAXNSR0IArs4c6QAAIABJREFUeF7snQnYXsPZxyeI2CO2oKEpgqKUNqittdSui4ooTS1FtbS0tTSf1q5pVa0pRZVPlYaitdS+tBKRamlV1RKfJZbEGklskch3/Q4T533ybOfcc94zz3n+c125SJ4zc+75zT33zH9mzjl95syZM8...

```

So yeah. need to add alt-text to the images we have on the notebooks
